docs/command_line_tools.md

```diff
@@ -199,6 +199,8 @@ The command can also be used non-interactively
 
 ```web3j generate-tests -i <Solidity java wrappers> -o <output path>```
 
+When passing in the path to the Solidity java wrappers class, the generated wrapper's classpath should be under a directory named java. If the classpath is `com.web3labs.GeneratedWrapper` then the directory structure should be as such: `/java/com/web3labs/GeneratedWrapper.java` where you would only pass the absolute path: `/home/user/java/`. Everything under the java directory is considered a classpath.
```

Comment:
This reads as if the wrapper must be placed in a folder called java? I don't think that's true?

Reply:
it is so if you generate wrappers with a structure such as org.com.contracts.generated.HelloWorld.java I have to tell the classloader where the full classname begins so if you want to generate the wrapper these need to be placed in a directory called java. When you generate the wrappers and pass is the path up to that java folder the classloader will consider everything under /java as a class name

Reply:
I understand you want to point it to the root folder for the first package part, i.e., you want to give it the folder in which the org folder lives in above example.

That's fine. My point is, does that folder need to be called "java"? If so, why?